_episodes/04-select.md

```diff
@@ -644,42 +652,9 @@ Here we can see why it was useful to transform these coordinates.  In
 ICRS, it is more difficult to identity the stars near the centerline
 of GD-1.
 
-So let's transform the results back to the GD-1 frame.
-Here's the code we used to transform the coordinates and make a Pandas
-`DataFrame`, wrapped in a function.
-
-~~~
-from gala.coordinates import reflex_correct
-
-def make_dataframe(table):
-    """Transform coordinates from ICRS to GD-1 frame.
-    
-    table: Astropy Table
-    
-    returns: Pandas DataFrame
-    """
-    skycoord = SkyCoord(
-               ra=table['ra'], 
-               dec=table['dec'],
-               pm_ra_cosdec=table['pmra'],
-               pm_dec=table['pmdec'], 
-               distance=8*u.kpc, 
-               radial_velocity=0*u.km/u.s)
-
-    gd1_frame = GD1Koposov10()
-    transformed = skycoord.transform_to(gd1_frame)
-    skycoord_gd1 = reflex_correct(transformed)
-
-    df = table.to_pandas()
-    df['phi1'] = skycoord_gd1.phi1
-    df['phi2'] = skycoord_gd1.phi2
-    df['pm_phi1'] = skycoord_gd1.pm_phi1_cosphi2
-    df['pm_phi2'] = skycoord_gd1.pm_phi2
-    return df
-~~~
-{: .language-python}
-
-Here's how we use it:
+We can use our `make_dataframe` function from episode 3 to transform the results back
+to the GD-1 frame. In addition to doing the coordinate transformation and reflex correction
+for us this function also compiles everything into a single object (a `DataFrame`) to make it easier to use. Note that because we put this code into a function, we can do all of this with a single line of code!
```

Comment:
```suggestion
for us, this function also compiles everything into a single object (a `DataFrame`) to make it easier to use. Note that because we put this code into a function, we can do all of this with a single line of code!
```